Changelog.md

```diff
@@ -2,6 +2,11 @@ OpenCore Changelog
 ==================
 #### v0.8.1
 - Improved `ExtendBTFeatureFlags` quirk on newer macOS versions, thx @lvs1974
+- ocvalidate: always print version compatibility info
+- Source code: codestyle fixes, cleanup and conversions
```

Comment:
Codestyle changes have no impact on functionality at all, hence no changelog is required.